src/apps/advanced-search/AdvancedSearchHeader.tsx

```diff
@@ -106,23 +111,24 @@ const AdvancedSearchHeader: React.FC<AdvancedSearchHeaderProps> = ({
   };
   const handleSearchButtonClick = () => {
     if (rawCql.trim() !== "" && !isFormMode) {
-      track("click", {
-        id: statistics.advancedSearchTerm.id,
-        name: statistics.advancedSearchTerm.name,
+      resetAndCollectPageStatistics({
+        ...statistics.advancedSearchCql,
         trackedData: rawCql
       });
+      updatePageStatistics({ waitTime: 1000 });
```

Comment:
I wonder why you set the wait time to `1000` here but `2500` and even `5000` elsewhere 🤔 .

Is it random or related to the context in some way?

Reply:
Good question! The wait times aren’t random, but they’re not strictly calculated either. It’s more of a feeling or intuition based on how long I think each step typically takes in context.